tests/test_query.py

```diff
@@ -99,24 +105,26 @@ def test_create_product_catalog(self):
         ]
         self.assertEqual(expected_names_and_versions, actual_names_and_versions)
 
-    def test_create_product_catalog_invalid_product_data(self):
+    '''def test_create_product_catalog_invalid_product_data(self):
```

Comment:
I'm assuming this function is commented only while this PR is in draft mode. Just noting here that it should be uncommented (or removed) before merging the PR.

Reply:
Correct. now the function is included